app/models/project.rb

```diff
@@ -43,7 +43,7 @@ class Project < ActiveRecord::Base
   ## TODO: This potential has locking issues
   validates_with UniqueForOwnerValidator
 
-  before_save :send_notifications
+  before_save :send_notifications, if: Proc.new { |project| project.persisted? }
```

Comment:
my bad on this I meant to only run on update can we do that instead of this proc?